com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml

```diff
@@ -0,0 +1,30 @@
+id: com.github.danrabbit.harvey
+runtime: org.gnome.Platform
+runtime-version: '3.28'
+sdk: org.gnome.Sdk
+command: com.github.danrabbit.harvey
+rename-icon: com.github.danrabbit.harvey
+tags: ['contrast', 'color']
```

Comment:
I would leave these out, this is AppData material, not standard in other apps as well.